tests/test_utils.py

```diff
@@ -4444,3 +4446,10 @@ def test_raises_unsupported_algorithm():
             "An error.", _Reasons.BACKEND_MISSING_INTERFACE
         )
     assert exc_info.type is UnsupportedAlgorithm
+
+
+def test_inspect_deprecated_module():
+    # Check if inspection is supported by _ModuleWithDeprecations.
```

Comment:
Can you add an assertion that `isinstance` check that the module is an `_ModuleWithDeprecations`? Otherwise this test becomes pointless when we removed `int_from_bytes`.